example/ToT/tasks/game24.py

```
@@ -0,0 +1,96 @@
+
+    def test_output(self, idx: int, output: str):
+        expression = \
+        output.strip().split('\n')[-1].lower().replace('answer: ', '').split('=')[0]
+        numbers = re.findall(r'\d+', expression)
+        problem_numbers = re.findall(r'\d+', self.data[idx])
+        if sorted(numbers) != sorted(problem_numbers):
+            return {'r': 0}
+        try:
+            # print(sympy.simplify(expression))
+            return {'r': int(sympy.simplify(expression) == 24)}
+        except Exception as e:
```

Ruff

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)